src/k8s/pkg/k8sd/features/cilium/network_test.go

```diff
@@ -390,4 +397,14 @@ func validateNetworkValues(g Gomega, values map[string]any, network types.Networ
 	g.Expect(values["ipam"].(map[string]any)["operator"].(map[string]any)["clusterPoolIPv6PodCIDRList"]).To(Equal(ipv6CIDR))
 	g.Expect(values["ipv4"].(map[string]any)["enabled"]).To(Equal((ipv4CIDR != "")))
 	g.Expect(values["ipv6"].(map[string]any)["enabled"]).To(Equal((ipv6CIDR != "")))
+
+	devices, exists := annotations.Get(apiv1_annotations.AnnotationDevices)
```

Comment:
nit: Maybe inline?

Reply:
I don't think we should inline since the returned value is used.

Reply:
oopsies! My bad!